### PR TITLE
Comms Agent Starts With A Paper Containing Comms Frequencies

### PR DESCRIPTION
**Pull request by eeSPee**
## About The Pull Request

Spawns a paper on Syndicate Lavaland Base contailing all the communications frequencies so that you don't need to wiki anymore, and a few tips, like how to disguise your voice and where to find records of all NT employees.

## Why It's Good For The Game

There is no IC explaination for Comms Agent to do their job or how to access frequencies

## Changelog

:cl:
add: added a paper for the syndicate comms agent containing all the frequencies, and how to use their equipment
/:cl:

### CODE REVIEW

File: code/datums/diseases/transformation.dm [hunk 1]
```diff
@@ -128,7 +128,7 @@
 		add_monkey(affected_mob.mind)
 	if(ishuman(affected_mob))
 		if(affected_mob && !is_monkey_leader(affected_mob.mind))
-			var/mob/living/carbon/monkey/M = affected_mob.monkeyize(TR_KEEPITEMS | TR_KEEPIMPLANTS | TR_KEEPORGANS | TR_KEEPDAMAGE | TR_KEEPVIRUS | TR_KEEPSE)
+			var/mob/living/carbon/monkey/M = affected_mob.monkeyize(TR_KEEPITEMS | TR_KEEPIMPLANTS | TR_KEEPVIRUS | TR_KEEPSE)
```
Comment (yorii) [inline code/datums/diseases/transformation.dm]: Why is this edit here?

Comment (eeSPee) [reply code/datums/diseases/transformation.dm]: Hm that shouldn't be there. Was messing with momke and must've saved by mistake.

**Comment by yorii** [comment]
Hahaha you deleted the entire file now :D

**Comment by eeSPee** [comment]
Alr8 it should be ok now.

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
There is a code file somewhere for adding in new papers with text that will show up, I think it's something like /obj/item/paper/fluff or something. That way you dont have to have the text in the map file. Also communications is spelt wrong.

**Comment by eeSPee** [comment]
> There is a code file somewhere for adding in new papers with text that will show up, I think it's something like /obj/item/paper/fluff or something. That way you dont have to have the text in the map file. Also communications is spelt wrong.

Like so?

**Review by PowerfulBacon — COMMENTED** [review]
Exactly like that, but communications is spelt with 2 Ms

**Comment by yorii** [comment]
> implemented: added the thing.
> fix: wording
> fix: it also changed momke
> tweak: separated it into fluff/

You should probably fix the changelog too, this is just a bunch of nonsense. The changelog is the actual information which gets presented to players ingame regarding changes done to the codebase, it should probably just contain:
> fix: added a paper with radio frequencies to the comms agent spawn

**Comment by eeSPee** [comment]
ok

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**